cmake/target.cmake

```diff
@@ -65,14 +65,19 @@ if (CMAKE_CROSSCOMPILING)
         set (HAS_POST_2038_EXITCODE "0" CACHE STRING "Result from TRY_RUN" FORCE)
         set (HAS_POST_2038_EXITCODE__TRYRUN_OUTPUT "" CACHE STRING "Output from TRY_RUN" FORCE)
 
+        # CMake < 3.13 doesn't respect same-name variables as values for options.
```

Comment:
If you don't know how to make such things with cmake you can ask colleagues. Will be fixed here https://github.com/ClickHouse/ClickHouse/pull/7063/commits/bf2a2659360423dc11601de74d79aad5a5701c35#diff-6f5216ab1e6a5d2983a9841326e86958R69